website/docs/r/independent_disk.html.markdown

```diff
@@ -48,7 +47,46 @@ The following arguments are supported:
 * `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful when connected as sysadmin working across different organisations
 * `vdc` - (Optional) The name of VDC to use, optional if defined at provider level
 * `name` - (Required) Disk name
-* `size` - (Required) - Size of disk in MB
+* `size` - (Optional; **Deprecated**) - Size of disk in MB
+* `size_in_bytes` - (Optional; *v2.5+*) size in bytes
 * `bus_type` - (Optional) - Disk bus type. Values can be: IDE, SCSI, SATA 
 * `bus_sub_type` - (Optional) - Disk bus subtype. Values can be: "IDE" for IDE. buslogic, lsilogic, lsilogicsas, VirtualSCSI for SCSI and ahci for SATA
-* `storage_profile` - (Optional) - The name of storage profile where disk will be created
\ No newline at end of file
+* `storage_profile` - (Optional) - The name of storage profile where disk will be created
```

Comment:
Small polishing needed:

1. Please remove the extra " - " after ") " in all five lines above. Should look like this:
 `size` - (Optional; **Deprecated**) Size of disk in MB

2. Capitalize first letter "S" after the ") " here as in other places:
 `size_in_bytes` - (Optional; *v2.5+*) size in bytes

Reply:
fixed